ui/lib/pki/addon/components/page/pki-certificate-details.ts

```diff
@@ -9,7 +9,7 @@ import { action } from '@ember/object';
 import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
 import errorMessage from 'vault/utils/error-message';
-import FlashMessageService from 'ember-cli-flash/services/flash-messages';
+import FlashMessageService from 'vault/services/flash-messages';
```

Comment:
anytime we see `FlashMessageService` we know it refers to the custom service file